magefile.go

```diff
@@ -1,3 +1,4 @@
+//go:build mage
```

Comment:
Since you are adding `//go:build mage` remove `//+build mage`.

Reply:
Fixed